src/features/indexer/indexerSlice.ts

```diff
@@ -86,12 +70,24 @@ export const indexerSlice = createSlice({
   extraReducers: (builder) => {
     builder.addCase(fetchIndexerUrls.fulfilled, (state, action) => {
       state.indexerUrls = action.payload;
+      if (!action.payload.length) {
+        state.errorText = i18n.t("orders.noIndexersFound");
+      }
     });
     builder.addCase(getFilteredOrders.fulfilled, (state, action) => {
       state.orders = action.payload;
```

Comment:
Now you're only using the result of the last indexer. Aren't you supposed to combine all the results?

Reply:
Not sure exactly what you're getting at here, but i've added a small edge case fix here. otherwise, if there are no indexers in state, and no indexers returning from the fetch, we will return this message.

Reply:
Is the following scenario possible?

Indexer 1 has order 1,2,4,5
Indexer 2 has order 1,3,4,6

Right now you only put the orders of indexer 2 in the store and you're missing order 2 and 5.

Reply:
`getIndexerOrders` in `IndexerRegistryApi` handles this correctly unless I'm having a massive brainfart.

Reply:
Ok I think I had multiple fulfilled `getFilteredOrders.fulfilled` fire at the same time. It seems to be fixed now. Sorry for the confusion!